pubsub/subscription.go

```diff
@@ -256,6 +256,14 @@ type SubscriptionConfig struct {
 	// FAILED_PRECONDITION. If the subscription is a push subscription, pushes to
 	// the endpoint will not be made.
 	Detached bool
+
+	// TopicMessageRetentionDuration indicates the minimum duration for which a message is retained
+	// after it is published to the subscription's topic. If this field is set,
+	// messages published to the subscription's topic in the last
+	// `TopicMessageRetentionDuration` are always available to subscribers.
+	//
+	// This field is set only in responses from the server and otherwise ignored.
```

Comment:
To be clear: this is an "output only" field? I know this isn't a proto field, but just want to clarify with https://google.aip.dev/203#output-only

Reply:
Yeah, this is an output only field, correspondig to https://github.com/googleapis/googleapis/blob/ba30d8097582039ac4cc4e21b4e4baa426423075/google/pubsub/v1/pubsub.proto#L761.

Should I explicitly add this to the comments here?

Reply:
> Should I explicitly add this to the comments here?

It's fine the way it is, but maybe it would be clearer to say `This is an output only field, meaning it will only appear in responses from the backend and will be ignored if sent in a request.` You could include the link too if you feel it would be helpful.

No change is really needed, just wanted to clarify for myself.